### PR TITLE
Fix Travis CI [DELIVERY-7091]

### PR DESCRIPTION
**Pull request by nathanegillett**
To get CI working for this project, this commit removes the Python 2.6 test environment due to the number of complications it introduces, corrects import locations in tests, and moves project requirements directly to setup.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,7 +21,13 @@ def get_requirements():
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
       ],
-      install_requires=get_requirements(),
+      install_requires=[
```
Comment (alexandrevicenzi) [inline setup.py]: is this change really necessary? I kinda like requirements.txt files.

File: test-requirements.txt [hunk 2]
```diff
@@ -4,6 +4,6 @@ koji
 configparser
 six > 1.10
 
-# Locked for python 2.7 support
+# Locked for python 2.6 support
 pytest-cov==2.5.1
-pytest==3.9.1
+pytest<3.3
```
Comment (alexandrevicenzi) [inline test-requirements.txt]: can you make this version lock conditional? Only for py 2.6 and a higher one for 2.7 or later? pytest 3 is already too old. If possible use latest 3.x or even 4.x for recent python.

File: setup.py (hunk 1 above)
Comment (nathanegillett) [reply setup.py]: Well, Travis can't seem to find the file on Python 2.6 or 3.6 for some reason so seems necessary for now, at least.

**Comment by nathanegillett** [comment]
@release-engineering/content-delivery would still like reviews of this even though it says it can be merged

**Comment by rbikar** [comment]
@nathanegillett It looks, that settings for Pull Request are not set in the way as in our other repos (required review, code owners etc...)

**Comment by nathanegillett** [comment]
> [...] settings for Pull Request are not set [...] (required review, code owners etc...)
Yeah, I'm not sure on those details so I don't want to set anything myself yet. Just looking for code review at the moment.

File: setup.py [hunk 3]
```diff
@@ -26,7 +21,7 @@ def get_requirements():
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
       ],
-      install_requires=get_requirements(),
+      install_requires=["koji", "rpm", "requests", "PyYAML", "simplejson"],
```
Comment (rohanpm) [inline setup.py]: Could we please avoid making this change?  Having the deps in a separate requirements.txt is useful in a few cases, it allows other tools (like github) to understand the project's dependencies and it makes it easier to patch them during RPM builds when that occasionally becomes needed.

You will probably find that adding "include requirements.txt" into MANIFEST.in would also fix whatever problem this was meant to resolve.  As I understand it, the issue is that current versions of setuptools include requirements.txt in the default list of files packaged for a project, but older versions don't, and Py 2.6 tests are using those older versions.

Comment (nathanegillett) [reply setup.py]: Looks like MANIFEST.in did fix the issue.
